examples/key-export/src/pages/index.tsx

```diff
@@ -0,0 +1,213 @@
+import Image from "next/image";
+import styles from "./index.module.css";
+import { TurnkeyApiTypes } from "@turnkey/http";
+import { IframeStamper } from "@turnkey/iframe-stamper";
+import * as React from "react";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+type TPrivateKey = TurnkeyApiTypes["v1PrivateKey"];
+
+const TurnkeyIframeContainerId = "turnkey-iframe-container-id";
+const TurnkeyIframeElementId = "turnkey-iframe-element-id";
+
+export default function ExportPage() {
+  const [privateKeys, setPrivateKeys] = useState<TPrivateKey[]>([]);
+  const [iframeStamper, setIframeStamper] = useState<IframeStamper | null>(
+    null
+  );
+  const [showPrivateKey, setShowPrivateKey] = useState<boolean>(false);
+  const [selectedPrivateKey, setSelectedPrivateKey] = useState<string | null>(
+    null
+  );
+
+  useEffect(() => {
+    const instantiateIframeStamper = async () => {
+      const stamper = new IframeStamper({
+        iframeUrl: process.env.NEXT_PUBLIC_EXPORT_IFRAME_URL!,
+        iframeContainerId: TurnkeyIframeContainerId,
+        iframeElementId: TurnkeyIframeElementId,
+        iframeStyle: "border: none; width: 600px; height: 200px;",
+      });
+
+      await stamper.init();
+      setIframeStamper(stamper);
+    };
+
+    if (!iframeStamper) {
+      instantiateIframeStamper();
+    }
+
+    return () => {
+      iframeStamper?.clear();
+    };
+  }, [iframeStamper]);
+
+  useEffect(() => {
+    getPrivateKeys();
+  }, []);
+
+  const getPrivateKeys = async () => {
+    const organizationId = process.env.NEXT_PUBLIC_ORGANIZATION_ID!;
+    const res = await axios.post("/api/getPrivateKeys", { organizationId });
+
+    setPrivateKeys(res.data.privateKeys);
+  };
+
+  const ExportIcon: React.FC = () => (
+    <svg viewBox="0 0 24 24" fill="none" width="12" height="12">
+      <path
+        d="M12 2L12 16M12 16L16 12M12 16L8 12M2 20H22"
+        stroke="#3f464b"
+        strokeWidth="2"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+
+  const formatCurve = (curve: string) => {
+    switch (curve) {
+      case "CURVE_SECP256K1": {
+        return "SECP256K1";
+      }
+      case "CURVE_ED25519": {
+        return "ED25519";
+      }
+      default: {
+        assertNever(curve);
+      }
+    }
+  };
```

Comment:
This could be pulled into its own file or function...but heh. Doesn't really hurt or help one way or the other since it's an example.